specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/ReplicationLinks.json

```diff
@@ -124,44 +124,6 @@
             "$ref": "./examples/ReplicationLinkGet.json"
           }
         }
-      },
```

Comment:
don't suggest to remove the operation if it is for fixing R4011- DeleteOperationResponses.  you can add missing response code to fix it. removing a operation is serious breaking change, not only for SDK.  let us know if there is other reason